src/router/utils/HelperFunctions.jsx

```diff
@@ -2,6 +2,24 @@ import axios from "axios";
 import Toast from "components/Toast";
 const userToken = localStorage.getItem("userToken");
 
+export const categories = async (videoDispatch) => {
+  try {
+    const response = await axios.get("/api/categories");
+    if (response.status === 200) {
+      videoDispatch({
+        type: "ALL_CATEGORIES",
+        payload: response.data.categories,
+      });
+    }
+  } catch (error) {
+    Toast({
+      type: "error",
+      message: "Oops! An error occurred.Try again later.",
+    });
+    console.log(error);
```

Comment:
remove the console log, and instead of writing a hardcoded error message, you can extract the exact message using "error.response.data.errors"